src/web/components/SharingPermission/SharingPermissionsTable.scss

```diff
@@ -1,74 +1,93 @@
-.sharing-permissions-table-header-container {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  margin-bottom: 1rem;
-}
+.sharing-permissions-table {
+  margin-top: 60px;
 
-.sharing-permissions-table h1 {
-  font-weight: 600;
-  font-size: 1.125rem;
-  line-height: 1.375rem;
-}
+  .sharing-permissions-table-header-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  .sharing-permissions-table h1 {
+    font-weight: 600;
+    font-size: 1.125rem;
+    line-height: 1.375rem;
+  }
+
+  .sharing-permissions-search-bar-container {
+    width: 190px;
+    display: flex;
+    align-items: center;
+    border-bottom: 2px solid var(--theme-search-border);
+    padding: 6px 2px;
+    justify-content: space-between;
+    ::placeholder {
+      color: var(--theme-search-border);
+      opacity: 1;
+    }
+  }
 
-.sharing-permissions-search-bar-container {
-  width: 190px;
-  display: flex;
-  align-items: center;
-  border-bottom: 2px solid var(--theme-search-border);
-  padding: 6px 2px;
-  justify-content: space-between;
-  ::placeholder {
+  .sharing-permission-search-bar-icon {
     color: var(--theme-search-border);
-    opacity: 1;
+    height: 1rem;
+    margin-left: 8px;
   }
-}
 
-.sharing-permission-search-bar-icon {
-  color: var(--theme-search-border);
-  height: 1rem;
-  margin-left: 0.5rem;
-}
+  .sharing-permission-actions {
+    display: flex;
+    align-items: center;
+  }
 
-.sharing-permissions-search-bar {
-  width: 100%;
-  border: none;
-  outline: none;
-  color: var(--theme-search-border);
-  font-weight: 400;
-  font-size: 12px;
-  line-height: 15px;
-}
+  .sharing-permissions-search-bar {
+    width: 100%;
+    border: none;
+    outline: none;
+    color: var(--theme-search-border);
+    font-weight: 400;
+    font-size: 12px;
```

Comment:
This is a good chance to clean this font-size and line-height up to rem.

Reply:
Yea, you are right, I updated height and margin, font-size and line-height to the scss files that involved in this PR